src/regridStates/regridStates.F90

```diff
@@ -39,15 +37,14 @@ program regridStates
  type(esmf_field), allocatable  :: fields_out(:) 
  type(esmf_routehandle)         :: regrid_route
  real(esmf_kind_r8), pointer    :: ptr_out(:,:)
+ 
+ integer :: ut
 
  real :: t1, t2, t3, t4
 
  ! see README for details of namelist variables.
- namelist /config/ dir_fix, res_atm_in, res_atm_out, fname_in, dir_in, &
-                   fname_out, dir_out, fname_out_rst, dir_out_rst, &
-                   variable_list, n_vars, missing_value, mask_type
+ namelist /config/ n_vars, variable_list, missing_value
```

Comment:
Do we need to set a default value here? 
variable_list=""

My run crushes unless it is specified, at least as empty string.

Reply:
There isn't really a default value, as it really depends on the use case. In the future, I'd like to add error checking in the setup routine to exit and report and error if required variables are not specified in the namelist. Would that be enough?

Reply:
Yes it does.

Reply:
I tried to add the function to check for presence of inputs quickly, but it wound up being a bit complicated. I will do this later though (or take care of it, when we call this from C++).